frontend/src/sass/elements/_date-picker.scss

```diff
@@ -0,0 +1,32 @@
+#main-content {
```

Comment:
if date picker is not going to be used, let's remove this.

Reply:
ok on 285